ui/app/mainui/AppMain.qml

```diff
@@ -1130,7 +1146,7 @@ Item {
         active: appMain.rootStore.mainModuleInst.sectionsLoaded
         sourceComponent: StatusEmojiPopup {
             height: 440
-            settings: localAccountSensitiveSettings
+            settings: appMainLocalSettings
```

Comment:
This is really confusing when reading the code. We could take `skinColor` and `recentEmojis` explicitly using separate properties.

Regarding the change we could use signals with change request or just treat those `skinColor` and `recentEmojis` as read/write and on change update the settings here in app main.

Reply:
Yup, will do